docs/_includes/display-component.html

```diff
@@ -1,14 +1,4 @@
-{% assign min = 50 %}
-{% assign max = 100000 %}
-{% assign diff = max | minus: min %}
-{% assign randomNumber = "now" | date: "%N" | modulo: diff | plus: min %}
 <div class="fd-tile docs-component docs-component__{{include.class}}">
-    <label class="fd-form-label fd-form-label--toggle docs-component__bg-toggle" for="{{ randomNumber }}" title="Change Background">
-        <span class="fd-toggle fd-toggle--xs">
-            <input class="toggle-bg fd-toggle__input" type="checkbox" name="" value="" id="{{ randomNumber }}">
-            <span class="fd-toggle__switch" role="presentation"></span>
-        </span>
```

Comment:
Why are we getting rid of the background toggle?

Reply:
What is the actual value of this?